src/main/kotlin/uk/gov/dwp/dataworks/services/TaskDeploymentService.kt

```
@@ -0,0 +1,136 @@
+
+       createService(ecs_cluster_name, user_name, ecsClient, albTargetGroupArn)
+
+        val userName: KeyValuePair = KeyValuePair.builder()
+                .name("user_name")
+                .value(user_name)
+                .build()
+
+        val emrClusterHostName: KeyValuePair = KeyValuePair.builder()
+                .name("emr_cluster_host_name")
+                .value(emr_cluster_host_name)
+                .build()
+
+        val chromeOverride: ContainerOverride = ContainerOverride.builder()
+                .name("headless_chrome")
+                .environment(userName)
+                .build()
+        val jupyterOverride: ContainerOverride = ContainerOverride.builder()
+                .name("jupyterHub")
+                .environment(userName, emrClusterHostName)
+                .cpu(jupyterCpu)
+                .memory(jupyterMemory)
+                .build()
+        val guacDOverride: ContainerOverride = ContainerOverride.builder()
+                .name("guacd")
+                .environment(userName)
+                .build()
+
+        val overrides: TaskOverride = TaskOverride.builder()
+                .containerOverrides(guacDOverride, chromeOverride, jupyterOverride)
+                .build()
+
+        val request: RunTaskRequest = RunTaskRequest.builder()
+                .cluster(ecs_cluster_name)
+                .launchType("EC2")
+                .overrides(overrides)
+                .taskDefinition("orchestration-service-ui-service")
+                .build()
+
+        println("Starting Task...")
+        try {
+            val response: RunTaskResponse = ecsClient.runTask(request)
+            println("response.tasks = ${response.tasks()}")
+        } catch (e: Exception) {
+            e.printStackTrace()
```

Same comment as above with suppressing exceptions